release/kubernetes-manifests.yaml

```diff
@@ -225,6 +225,33 @@ spec:
     port: 8080
     targetPort: 8080
 ---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+ - adservice.yaml
+ - cartservice.yaml
+ - checkoutservice.yaml
+ - currencyservice.yaml
+ - emailservice.yaml
+ - frontend.yaml
+# - loadgenerator.yaml # During development, the loadgenerator module inside skaffold.yaml will be used.
+ - paymentservice.yaml
+ - productcatalogservice.yaml
+ - recommendationservice.yaml
+ - redis.yaml
+ - shippingservice.yaml
+# components:
+# - ../kustomize/components/cymbal-branding
+# - ../kustomize/components/google-cloud-operations
+# - ../kustomize/components/memorystore
+# - ../kustomize/components/network-policies
+# - ../kustomize/components/service-accounts
+# - ../kustomize/components/spanner
+# - ../kustomize/components/container-images-tag
+# - ../kustomize/components/container-images-tag-suffix
+# - ../kustomize/components/container-images-registry
+# - ../kustomize/components/native-grpc-health-check
+---
```

Comment:
**Issue:** This `Kustomization` resource should not exist in this file. Working on fix now.

Reply:
Fixed!

Reply:
Fixed!